src/sample/complete/TicketService.java

```diff
@@ -1,29 +1,44 @@
 package sample.complete;
 
 import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Random;
+import java.util.stream.IntStream;
 
 public class TicketService {
+
+	private static Map<Integer, Ticket> tickets;
+
 	public static void main(String[] args) {
 
-		long unixTimestamp = Instant.now().getEpochSecond();
-		System.out.println("-".repeat(80));
+		tickets = new HashMap<>();
 
-		// empty
-		Ticket ticketEmpty = new Ticket();
-		System.out.println(ticketEmpty);
-		System.out.println("-".repeat(80));
+		// generate 10 tickets
+		IntStream.rangeClosed(1, 10)
+				.forEach(i -> {
```

Comment:
what is i? an id, a ticket or something else? please give a more understandable name